src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackTraceExporter.cs

```diff
@@ -400,6 +400,23 @@ public void Dispose()
 
     private const int BUFFER_SIZE = 65360; // the maximum ETW payload (inclusive)
 
+    private static readonly string INVALID_SPAN_ID = default(ActivitySpanId).ToHexString();
```

Comment:
Probably not part of this PR, but snake case (all caps) is rarely used.

Reply:
Yes, this project should use the standard C# naming conventions.